src/app/page/cabinet/cabinet.component.html

```diff
@@ -93,19 +93,21 @@ <h1 class="modal-title" id="exampleModalTitle">Choose deck</h1>
                 <th>Add</th>
               </tr>
             </thead>
-            <tbody>
-              <tr *ngFor="let deck of categoryDecks">
-                <td>{{deck.name}}</td>
-                <td>{{deck.description}}</td>
-                <td>
-                  <star-rating-comp [starType]="'svg'" [showHalfStars]="true" [readOnly]="true"
-                                    [rating]="deck.rating"></star-rating-comp>
-                </td>
-                <td><img data-dismiss="modal" src="../../../assets/images/add.png" title="Add"
-                         (click)="addDeckToCourse(deck)" class="deck-img"></td>
-              </tr>
+            <tbody *ngFor="let deck of categoryDecks">
```

Comment:
Table should contain only one body. Is is the case?

Reply:
Sorry, what do you mean? Doesn't it contain only one body?

Reply:
Tell me, please, how can I fix it then? It doesn't allow me to use two *ngIf statements in one tag.

Reply:
<ng-template> is the first idea.

But taking into account that you want to show only some elements of the list, possibly the list should be filtered in advance.

Reply:
Okay, thank you! I'm going to filter it and then I'll commit my changes.

Reply:
@antonbakalets I'm sorry but it is impossible to filter this array before displaying it because its elements' property "hidden" is changed within this loop... So for now I have no idea how to fix that. In any case it wworks well...